### DISCUSSION TITLE
How can i Retrive Single column data

### DISCUSSION THREAD

**Discussion by RajNandigalla** [strapi/strapi#6977] [Legacy Topics]
is it possible to achieve it using strapi ?

`db.posts.distinct('category')`
that return an Array of Categories `[ 'Category1' , 'Category2'..............................]`

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.
